File: app/src/main/java/com/arjanvlek/oxygenupdater/help/HelpActivity.java
package com.arjanvlek.oxygenupdater.help;

import android.os.Bundle;
import android.view.MenuItem;

import com.arjanvlek.oxygenupdater.R;
import com.arjanvlek.oxygenupdater.views.SupportActionBarActivity;

public class HelpActivity extends SupportActionBarActivity {

	@Override
	public void onCreate(Bundle savedInstanceSate) {
		super.onCreate(savedInstanceSate);
		setContentView(R.layout.activity_help);
	}

	@Override
	public void onBackPressed() {
		finish();
	}

	@Override
	public boolean onOptionsItemSelected(MenuItem item) {
		// Respond to the action bar's Up/Home button
		if (item.getItemId() == android.R.id.home) {
			finish();
			return true;
		}

		return super.onOptionsItemSelected(item);
	}

}

File: app/src/main/java/com/arjanvlek/oxygenupdater/help/HelpActivity.kt
package com.arjanvlek.oxygenupdater.help

import android.os.Bundle
import android.view.MenuItem
import com.arjanvlek.oxygenupdater.R
import com.arjanvlek.oxygenupdater.views.SupportActionBarActivity

class HelpActivity : SupportActionBarActivity() {

    public override fun onCreate(savedInstanceSate: Bundle?) {
        super.onCreate(savedInstanceSate)
        setContentView(R.layout.activity_help)
    }

    override fun onBackPressed() = finish()

    /**
     * Respond to the action bar's Up/Home button
     */
    override fun onOptionsItemSelected(item: MenuItem) = if (item.itemId == android.R.id.home) {
        finish()
        true
    } else {
        super.onOptionsItemSelected(item)
    }
}
